src/components/artemis/documentation/latest/management-console.html

```diff
@@ -226,4 +226,4 @@ <h3 id="status-logging"><a class="anchor" href="#status-logging"></a><a class="l
 </div>
 </div>
 </body>
-</html>
```

Comment:
This is an unintended whitespace change. Please ignore/undo.

Reply:
Can you fix the commit and do a `push -f`?